src/test/java/com/twilio/TwilioTest.java

```diff
@@ -64,6 +63,23 @@ public void testSetAuthTokenNull() {
         fail("AuthenticationException was expected");
     }
 
+    @Test
+    public void testUserAgentExtensions() {
+        Twilio.setUsername("UserName");
+        Twilio.setPassword("Password");
+        Twilio.setUserAgentExtensions(Arrays.asList("ce-appointment-reminders/1.0.0", "code-exchange"));
+        TwilioRestClient twilioRestClient = Twilio.getRestClient();
+        assertNotNull(twilioRestClient);
+    }
+
+    @Test
+    public void testUserAgentExtensionsEmpty() {
+        Twilio.setUsername("UserName");
+        Twilio.setPassword("Password");
+        Twilio.setUserAgentExtensions(Arrays.asList());
+        TwilioRestClient twilioRestClient = Twilio.getRestClient();
+        assertNotNull(twilioRestClient);
+    }
```

Comment:
There's no assertion here that ensures the rest client actually contains the extensions. Think the test should call `request` and asset that the call to `reliableRequest` contains a request with the extensions.